integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java

```diff
@@ -413,31 +416,31 @@ void testDomainK8sEventsScalePastMaxAndChangeIntrospectVersion() {
     OffsetDateTime timestamp = now();
     try {
       logger.info("Scaling cluster using patching");
+      assertFalse(scaleCluster(cluster1Name, domainNamespace3, 3), "failed to scale cluster via patching");
       String patchStr
```

Comment:
There is a utility to update the introspector version, instaed of hard-coding the version to 12345

Reply:
done